src/manifest.ts

```diff
@@ -87,6 +97,40 @@ export async function enableOrDisableAdvanceServiceInManifest(serviceId: string,
   await writeManifest(manifest);
 }
 
+/**
+ * Reads the remote appsscript.json manifest file.
+ * @returns {Promise} A promise to get the remote manifest file as object.
+ * @see https://developers.google.com/apps-script/concepts/manifests
+ */
+export async function readRemoteManifest() : Promise<Manifest>  {
+  await checkIfOnline();
```

Comment:
Can you reuse more of the `clasp pull` method?

Refactor `clasp pull` so we use a common method.
We shouldn't see `await script.projects.getContent({scriptId});`.

```ts
const content = await getProjectContent(scriptId);
const files = content.data.files || [];
const manifest = files.find((file) => file.name === PROJECT_MANIFEST_BASENAME);
```

Also, Apps Script forces manifests to be valid JSON so we don't need to try JSON parse.